panels_media_pbject/main.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright (C) Foundry 2021
+#
+
+import argparse
+import datetime
+import os
+import signal
+import sys
+import flix as flix_api
+
+
+def get_panels_with_media_objects(show_id, episode_id, sequence_id, revision_id):
+
+    # Get selected revision
+    revision = flix_api.get_sequence_revision_by_id(
+        show_id, episode_id, sequence_id, revision_id)
+
+    if revision == None:
+        print('Revision not found.')
+        sys.exit(1)
+
+    # Get all panels in the sequence revision
+    panels = flix_api.get_sequence_revision_panels(
+        show_id, episode_id, sequence_id, revision_id)
+
+    if panels == None:
```

Comment:
Cannot check equality with `None`
```suggestion
    if panels is None:
```